resources/plugin.yml

```diff
@@ -57,6 +57,14 @@ permissions:
             towny.wild.*: true
             towny.claimed.*: true
             towny.admin.nation_zone: true
+            towny.meta.modify: true
```

Comment:
nodes should reflect their commands which I believe are  towny.command.townyadmin.plot.meta and towny.command.townyadmin.town.meta. They should be included as child nodes of towny.command.townyadmin.town.* and towny.command.townyadmin.plot.*

Reply:
The two new nodes should then be added to the com.palmergames.bukkit.towny.permissions.PermissionNodes class.